packages/postgres-database/src/simcore_postgres_database/models/services.py

```diff
@@ -160,3 +188,36 @@
         "key", "version", "gid", "product_name", name="services_access_pk"
     ),
 )
+
+
+#
+# services_latest table:
+#   Keeps latest version of every service (key)
+#
+
+services_latest = sa.Table(
+    "services_latest",
```

Comment:
to be honnest, I am a bit surprised by this table. does this really bring a lot? I mean, both the director-v0 and the catalog are already caching their respective data. out of the blue this looks like a duplicate of information that already exists. Is there no way of making the postgres auto-generate this maybe?